index.js

```diff
@@ -43,7 +44,7 @@ module.exports = (moduleObj, options) => {
 		...options
 	};
 
-	const cwd = path.dirname(moduleObj.filename);
+	const cwd = path.dirname(await findUp('package.json'));
```

Comment:
You are implicitly using `process.cwd()` for the start location here, which is not correct. `process.cwd()` is wherever the process was started (can be a completely different directory). I think you'll want to set the `cwd` option in `findUp` to `app.getAppPath()`, but not sure. Please do some more thorough testing.